### DISCUSSION TITLE
Logo for Organization

### DISCUSSION THREAD

**Discussion by Nonethousand** [AspectLinux/aspectlinux.github.io#12] [Ideas]
I noticed that there was no logo for this organization, so I made a simple one.

I also tried to stay true to the original logo which is [here](https://avatars.githubusercontent.com/u/94648612?s=200&v=4)

![AspectLogo](https://user-images.githubusercontent.com/78574005/214686507-1a14565a-e81a-499b-b692-ff80e78278ca.svg)

And here is a orange and white recolor because the above one kinda looks like the Photoshop logo

![AspectLogoOrange](https://user-images.githubusercontent.com/78574005/214693965-a818e534-81f1-45f3-afd8-ec46bb23ca45.svg)

**Comment by PPPDUD** [comment]
Looks more like Photoshop than an OS logo - no offense.

**Reply by Nonethousand** [reply]
Well what could I change about it?

Edit: Just looked up the photoshop logo
and yeah they do look kinda similar

**Comment by PPPDUD** [comment]
Maybe refrain from using black and blue, instead using something like
orange + white?

**Reply by Nonethousand** [reply]
I used black and blue because the original logo was those colors. I can change them, but I'll keep the blue one in the OP just in case

**Comment by PPPDUD** [comment]
Good idea. Mind if I submit a mockup too?

**Reply by Nonethousand** [reply]
Sure.
And in the future could you reply instead of creating a new comment?

**Comment by PPPDUD** [comment]
I would, but I'm writing this via Gmail, so there's not an option to do
that.

**Reply by Nonethousand** [reply]
Oh okay, didn't realize that.

**Comment by udscminidisc** [comment]
I love it!
